.circleci/config.yml

```diff
@@ -0,0 +1,51 @@
+version: 2.1
+
+#
+# Variables
+#
+
+project_directory: &project_directory ~/project
+
+#
+# Jobs
+#
+
+jobs:
+  test:
+    working_directory: *project_directory
+    macos:
+      xcode: 11.5.0
+    resource_class: large # Uses the "large" macOS executor
```

Comment:
[nit] Consider cheaper?

Reply:
Totally.  I wanted to compare the two (especially while we are in a trial) :).  The results seem to indicate that the difference in time is minimal (a few seconds, maybe).